packages/common/src/converter/payload-converter.ts

```diff
@@ -140,3 +149,75 @@ export class BinaryPayloadConverter extends AsyncFacadePayloadConverter {
     return content.data as any;
   }
 }
+
+/**
+ * Converts between protobufjs Message instances and serialized Protobuf Payload
+ */
+export class ProtobufPayloadConverter extends AsyncFacadePayloadConverter {
+  public encodingType = encodingTypes.METADATA_ENCODING_PROTOBUF;
+
+  constructor(protected protobufClasses?: Record<string, unknown>) {
```

Comment:
the attribute itself shouldn't be optional
```suggestion
  constructor(protected protobufClasses: Record<string, unknown> = {}) {
```